source/server/management/performance-tweaks.rst

```diff
@@ -87,16 +106,34 @@ lower this interval, then change it in the servers ``global.conf`` file:
 
 .. code-block:: none
 
-  world {
+    world {
         # The auto-save tick interval used when saving global player data.
         # Set to 0 to disable. (Default: 900) Note: 20 ticks is equivalent to 1 second.
         auto-player-save-interval=900
 
         # The auto-save tick interval used to save all loaded chunks in a world.
         # Set to 0 to disable. (Default: 900) Note: 20 ticks is equivalent to 1 second.
         auto-save-interval=900
-        }
+    }
 
 Reducing this interval increases the load on your server's CPU and storage, but reduces the data loss that might occur
 if the server locks up or the power fails. Conversely, increasing the auto-save interval reduces the load on the
 hardware, but at the expense of increasing the amount of in-game progress that could be lost in case of server failure.
+
+Realtime
+========
+
+Enabling this will not actually improve performance, but instead will improve your experience a lot when tick rate is 
```

Comment:
This should start with what it *does*, and leave what it *does not do* to the end.

Reply:
✅